test/synthetic/test_synthetic_data.py

```diff
@@ -50,10 +50,10 @@ def test_perfect_LF(self) -> None:
         L = lf_applier.apply(data)
 
         # Get accuracies
-        Ld = L.todense()
+        # TODO: Replace this with a generic utility function
         Y = np.array([d.y for d in data])
-        accs = Ld.T @ Y / self.n
-        self.assertEqual(accs[0], 1.0)
+        acc = np.where(L[:, 0].toarray().reshape(-1) == Y, 1, 0).sum() / self.n
```

Comment:
can we just use https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html?